app/models/library_entry.rb

```diff
@@ -193,9 +193,34 @@ def kind
     media.trending_vote(user, 1.0) if status_changed?
   end
 
-  after_commit ->() { sync_entry(:create) }, on: :create
-  after_commit ->() { sync_entry(:update) }, on: :update
-  after_destroy ->() { sync_entry(:delete) }
+  after_commit(on: :create) do
+    sync_entry(:create) # mal exporter
+  end
+
+  after_commit(on: :update) do
+    sync_entry(:update) # mal exporter
+  end
+
+  after_create do
+    # Stat STI
+    case media_type
```

Comment:
I'd prefer if you used `kind` for this (returns a symbol `:anime`, `:manga`, etc.), since `media_type` is deprecated and slated for eventual removal